.github/workflows/build.yml

```diff
@@ -0,0 +1,131 @@
+on:
+  push:
+    branches:
```

Comment:
In the Travis.yml the branches are
    - develop
    - /^APPS-.*$/
    - /support\/.*/